components/board/numato/board.go

```diff
@@ -237,9 +237,12 @@ func (b *numatoBoard) StreamTicks(ctx context.Context, interrupts []string, ch c
 }
 
 // AnalogByName returns an analog pin by name.
-func (b *numatoBoard) AnalogByName(name string) (board.Analog, bool) {
+func (b *numatoBoard) AnalogByName(name string) (board.Analog, error) {
 	ar, ok := b.analogs[name]
-	return ar, ok
+	if !ok {
+		return ar, errors.Errorf("can't find AnalogReader (%s)", name)
```

Comment:
please use the "fmt" package since https://github.com/pkg/errors is an archived go library. it's okay if a file already uses this package, but we'd rather use packages that are maintained, especially for errors. So let's try to use golang building packages for error printing.